WebContent/js/components/dialogs/datasets/CreateDatasetDialog.js

```diff
@@ -73,6 +73,8 @@ export default class CreateDatasetDialog extends React.Component {
             recordFormat: PRESETS.get('JCL').recordFormat,
             blockSize: PRESETS.get('JCL').blockSize,
             recordLength: PRESETS.get('JCL').recordLength,
+            disableSubmit: false,
```

Comment:
Since it inits to `name: ''`, shouldn't `disableSubmit` be `true` to start?

Reply:
done